using-timescaledb/continuous-aggregates.md

```diff
@@ -16,148 +17,198 @@ every refresh. Only new and/or invalidated data will be calculated.  Since this
+
+If you, however, want to just get partially aggregated data and not
+include recent data that has yet to be materialized, you can set the option
+`timescaledb.materialized_only` to `true` using `ALTER VIEW`:
+
+```sql
+ALTER VIEW conditions_summary_hourly SET (
+    timescaledb.materialized_only = true
+);
+```
+
+>:TIP: You can find the names of all the materialized hypertable by
+>querying `timescaledb_information.continuous_aggregates`.
+>
+> ```sql
+> postgres=# SELECT view_name, materialization_hypertable
+> postgres-#   FROM timescaledb_information.continuous_aggregates;
+>          view_name         |            materialization_hypertable             
+> ---------------------------+---------------------------------------------------
+>  conditions_summary_hourly | _timescaledb_internal._materialized_hypertable_30
+>  conditions_summary_daily  | _timescaledb_internal._materialized_hypertable_31
+> (2 rows)
+> ```
+
+Each continuous aggregate uses the *bucket width* of the `time_bucket`
+(given by the parameter `bucket_width`) function in the view
+definition to decide what to update. Time-series data is typically
+ordered, so it is usually the last bucket that gets most of the
+updates. Recomputing the last bucket when new data arrives negates many
+of the benefits of using continuous aggregation. Older buckets rarely
+get updated and are usually aggregated only once.
+
+As a result, the materialization lags behind by one bucket and up to an additional `refresh_interval` as a
+result of the job not running continuously. However, you can change
+this lag to either increase it (or decrease it) by setting the value
+for the `timescaledb.refresh_lag` option of the view and the
+background worker will attempt to materialize everything before
+`refresh_lag` + `bucket_width`.
+
+For example, using `conditions_summary_hourly` above, the bucket width
+is 1 hour, so if we set the `refresh_lag` to 1 hour, the
+materialization will be 2 hours behind the current time.
+
+```sql
+ALTER VIEW conditions_summary_hourly SET (
+  timescaledb.refresh_lag = '1 hour'
+);
+```
+
+Lowering the `refresh_lag` parameter means that the materialization
+follows the raw data more closely, but can cause write amplification
+that might degrade insert performance. The `refresh_lag` can also be
+set to to negative values. For example, setting `refresh_lag` to
+`-bucket_width` will make the continuous aggregate materialize new
+data as soon as it is available.
+
+The `refresh_lag` parameter controls the balance between the
+on-the-fly aggregation and pre-computed aggregation when querying the
+continuous aggregate. A lower `refresh_lag` will reduce the amount of
+on-the-fly aggregation by more aggressive materialization and a higher
+`refresh_lag` will increase the amount of on-the-fly aggregation by
+not materializing as agressively.
+
+>:WARNING: Changing the value of `refresh_lag` is not recommended and
+>should be avoided since it rarely is necessary.
```

Comment:
why is changing refresh_lag not recommended, i think this warning should be removed

Reply:
Because it is not needed now that we have real-time aggregates?

Reply:
Not needed and not recommended are very different though. If you want to add that its not needed then this should be a TIP and state that setting it won't be needed because of real-time aggregation

Reply:
I just transformed it into a tip.